### ISSUE TITLE
App doesn't load - Windows 10/Chrome latest version

### ISSUE THREAD

**Issue by leofrish** [tjmonsi/chi2016-program#5]
Click on the Conference App tab and the system simply says "Loading..."

**Comment by tjmonsi** [comment]
That's weird. I am also using Windows 10/Chrome version. It might not be loading fast enough? You can refresh it if it doesn't work

**Comment by leofrish** [comment]
Thanks Toni-Jan. It's intermittent and it worked almost all day today, but
it was the second time in two days that I had a problem.

Thanks!

On Mon, May 9, 2016 at 10:00 PM, Toni-Jan Keith Monserrat <
notifications@github.com> wrote:

> That's weird. I am also using Windows 10/Chrome version. It might not be
> loading fast enough? You can refresh it if it doesn't work
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/tjmonsi/chi2016-program/issues/5#issuecomment-218059486>
>



-- 
*Design to Fail - Presumptive Design <http://www.presumptivedesign.com>*



* <http://www.phaseiidesign.com/>*

LinkedIn Profile <https://www.linkedin.com/pub/leo-frishberg/0/41/b51>

**Comment by tjmonsi** [comment]
Yeah i figured that was the problem. The connection to the database, which
is also used by the mobile app but different from the full sched, is a bit
finicky... Sorry about that. We'll try to improve the service though but i
can't promise much
